include/sys/dmu.h

```diff
@@ -902,6 +902,8 @@ extern uint_t zfs_max_recordsize;
  */
 void dmu_prefetch(objset_t *os, uint64_t object, int64_t level, uint64_t offset,
 	uint64_t len, enum zio_priority pri);
+void dmu_prefetch_by_dnode(dnode_t *dn, int64_t level, uint64_t offset,
+	uint64_t len, enum zio_priority pri);
```

Comment:
Can you add an `EXPORT_SYMBOL()` line in the matching source file for the new `_by_dnode()` functions.  This isn't needed by ZFS directly but I'd like to make these new kmod symbols available to the Lustre kmods.  They already take advantage of the `_by_dnode` symbols for performance and would benefit from this.